jobs/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: Open positions
+excerpt: "We want You!"
+author: peter_krauss
+search_omit: true
+---
+<ul class="post-list">
+{% for post in site.categories.jobs %}
+  <li><article><a href="{{ site.url }}{{ post.url }}">{{ post.title }}</a></article></li>
+{% endfor %}
+</ul>
+
+## About us
+Upday is a start-up based in the heart of Europe’s most exciting city, Berlin. Used by millions of users in several European countries, our app is pre-installed on the -1 homescreen page of Samsung’s flagship devices.
+
+We are a passionate, diverse and international team, always striving to improve, curious, motivated and keen to go the extra mile. We have a strong culture of collegiality and an extraordinary team spirit where we are not afraid to celebrate success from individual, team to company achievements, anniversaries, birthdays, weddings/babies–you name it, we celebrate it.
+
+**Some reasons why you wanna work with us**
+
+* It’s an opportunity to work with passionate, curious people with a constant strive to improve and go for the extra mile
+* We have a strong culture of collegiality and an extraordinary team spirit that we are happy to share with you and are proud of
+* We provide personal growing support: from blog posts to tech talks and knowledge sharing (http://upday.github.io/).
```

Comment:
in the same time, the user is already on the website... it doesn't make much sense to have the link in general...